pom.xml

```diff
@@ -54,6 +58,10 @@
 			<artifactId>lombok</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate.validator</groupId>
```

Comment:
You can add a spring boot validation starter instead of a concrete implementation.